packages/devextreme/js/__internal/ui/html_editor/ui/aiDialog.ts

```diff
@@ -480,6 +484,7 @@ export default class AIDialog extends BaseDialog<AIDialogResult> {
     this._refreshToolbarItems();
     this._refreshLoadIndicator();
     this._refreshInformer();
+    this._refreshDialogAria();
```

Comment:
Does title depend on state? 
If not, isn't it better to do it single time in popup's `onContentReady`?

Reply:
No, we can't.

Now that I've looked deeper into the code, I can see that when the state changes, there is a constant update of ToolbarItems: https://github.com/DevExpress/DevExtreme/pull/29833/files#diff-72ca008cf68f852ae0f88cd7cb34d2c59825c068ad1cb7e2c332746cd1f98a86R491 

This causes the Toolbar Items to re-render every time the state changes, which means that the header that was added at the beginning - no longer exists.

In good conscience, we shouldn't design a component this way. But the current redraws have almost no impact on performance.